src/modules/developer_tools.json

```diff
@@ -0,0 +1,250 @@
+            "name": "unpair",
+            "summary": "Disconnect Firebolt as a web socket client from the web socket that corresponds to the given developer tool ID",
+            "tags": [],
+            "params": [
+                {
+                    "name": "developerToolId",
+                    "summary": "The ID of the DeveloperTool from which to disconnect",
+                    "required": true,
+                    "schema": {
+                        "type": "string",
+                        "format": "uuid"
+                    }
+                }
+            ],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "const": null
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [
+                        {
+                            "name": "developerToolId",
+                            "value": "ff6340fb-8576-494c-97c6-e60e9a4eeba9"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": null
+                    }
+                }
+            ]
+        },
+        {
+            "name": "onDeveloperToolUp",
+            "tags": [
+                {
+                    "name": "event"
+                }
+            ],
+            "summary": "Listen to the developerToolUp event",
+            "params": [
+                {
+                    "name": "listen",
+                    "required": true,
+                    "summary": "whether to listen, or stop listening",
+                    "schema": {
+                        "type": "boolean"
+                    }
+                }
+            ],
+            "result": {
+                "name": "value",
+                "schema": {
+                    "oneOf": [
+                        {
+                            "$ref": "#/components/schemas/DeveloperTool"
+                        },
+                        {
+                            "$ref": "https://meta.comcast.com/firebolt/types#/definitions/ListenResponse"
+                        }
+                    ]
+               }
+            },
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [
+                        {
+                            "name": "listen",
+                            "value": true
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": {
+                            "id": "ff6340fb-8576-494c-97c6-e60e9a4eeba9",
+                            "name": "sideloader"
+                        }
+                    }
+                }
+            ]
+        },
+        {
+            "name": "onDeveloperToolDown",
```

Comment:
Does ripple actually get an event when a tool goes down? I thought only the DNS cache on the device gets updated and ripple would have to query it.